### PR TITLE
SWAN-18: Create Computig provider collateral balace check

### PR DESCRIPTION
**Pull request by Arash-Swan**
Create Computig provider collateral balace check

### CODE REVIEW

**Review by DanielJiangCloud — DISMISSED** [review]
LGTM

File: test/api/test_cp.py [hunk 1]
```diff
@@ -46,3 +46,72 @@ def test_failed_api_response(self):
...
+            # Call the function with a valid computing provider address
+            result = get_collateral_balance("0x1234abcd")
+
+            # Assert that the response is as expected
+            assert result == {
+                "status": "success",
+                "message": "Successfully retrieved collateral balance",
+                "data": {"balance": 100},
+            }
+    # NOTE: do not uncomment before ApiClient changes
```
Comment (pluto-workhub) [inline test/api/test_cp.py:83]: are we planing to chagne the API Client ?

**Comment by pluto-workhub** [comment]
RC

File: test/api/test_cp.py (hunk 1 above)
Comment (Arash-Swan) [reply test/api/test_cp.py:83]: No, I need to make changes on TEST by tokens

Comment (pluto-workhub) [reply test/api/test_cp.py:83]: Got it. then lgtm

**Pull request merged**